lib/realtime/helpers.ex

```diff
@@ -103,6 +103,54 @@ defmodule Realtime.Helpers do
     end
   end
 
+  @doc """
+  Gets a short node name from a node name when a node name looks like `realtime-prod@fdaa:0:cc:a7b:b385:83c3:cfe3:2`
+
+  ## Examples
+
+      iex> node = Node.self()
+      iex> Realtime.Helpers.short_node_id_from_name(node)
+      "localhost"
+
+      iex> node = :"realtime-prod@fdaa:0:cc:a7b:b385:83c3:cfe3:2"
+      iex> Realtime.Helpers.short_node_id_from_name(node)
+      "83c3cfe3"
+
+      iex> node = :"pink@127.0.0.1"
+      iex> Realtime.Helpers.short_node_id_from_name(node)
+      "localhost"
+
+      iex> node = :"pink@10.0.1.1"
+      iex> Realtime.Helpers.short_node_id_from_name(node)
+      "10.0.1.1"
+  """
+
+  @spec short_node_id_from_name(atom()) :: String.t()
+  def short_node_id_from_name(name) when is_atom(name) do
+    [_, ip] = name |> Atom.to_string() |> String.split("@", parts: 2)
+
+    {:ok, ip} = ip |> handle_nohost() |> String.to_charlist() |> :inet.parse_address()
```

Comment:
But this only works for IP addresses. the node with hostname, for example, node@localhost will produce an error

Reply:
Fixed!